### ISSUE TITLE
C++20 Coroutine support

### ISSUE THREAD

**Issue by malik1988** [wxWidgets/wxWidgets#25009]
### Description

Need Coroutine support for wx to simplify callbacks.
 Is it scheduled?

**Comment by oneeyeman1** [comment]
What you should do is build wx with C++20 support and then build your application with it.
Then you will be able to use whatever C++20  you want.

**Comment by malik1988** [comment]
> What you should do is build wx with C++20 support and then build your application with it.
> Then you will be able to use whatever C++20  you want.

It's not that easy from my site.

- how to turn all the evenloops into co_awaitable?
- how to safely manage local context when binding co tasks?
- is there any examples?

**Comment by vadz** [comment]
No, it is not scheduled, but would be great to have. Of course, this would be a lot of work and it would need to be discussed first, to determine, at least roughly, the API that we won't provide, but any contributions would be welcome.
